Automattic/wp-calypso
# BBE: add 5 new page types

Fixes https://github.com/Automattic/martech/issues/2139

## Proposed Changes

Adds 5 new pages to the BBE page picker step, with its associated strings and images.

- Careers
- Events
- Donate
- Newsletter
- Case Studies

### Screenshots

<img width="970" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/7b29a0c5-30ce-4f14-bae8-47a874481555">

| Page | Page Picker step | Website Content Step |
|--------|--------|--------|
| Careers | <img width="291" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/498fc0ff-682a-4699-a388-3dc990d9bf9b"> | <img width="769" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/58ecd2b0-2ca8-4469-8467-48af557bde73"> |
| Events | <img width="281" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/fb63f5c0-a07d-4ed1-ab53-4206dad19af6"> | <img width="791" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/0091f7b6-2c11-4811-ae54-77fa6dd623da"> |
| Donate | <img width="302" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/225c79ac-29b9-4177-ac4e-09d01e5ce728"> | <img width="751" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/ea7f218f-1d9d-4681-a3cf-fbd7c115a19d"> |
| Newsletter | <img width="299" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/06725961-da31-4c98-9f2e-4bab9de1ce1b"> | <img width="739" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/b46695d7-016c-4f54-8bb1-800bfc48c29a"> |
| Case Studies | <img width="285" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/930d00f4-90a3-4ff9-8e63-1c17cff5e627"> | <img width="780" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/0cb7acc6-dab9-4cc7-a11d-53bcd2bbe3ba"> |

## Testing Instructions



* Go to `/start/do-it-for-me` and proceed through the flow steps till you reach the page picker step.
* Confirm that the new pages mentioned above are visible and that the title, description and thumbnail match the screenshots.
* Select the new pages and complete the purchase.
* On the website content submission form, confirm that the description for the "Donate" and "Newsletters" page is different from the one on the page picker step and that it matches the screenshots.

## Pre-merge Checklist
